SConstruct

```diff
@@ -2490,6 +2502,10 @@ def doConfigure(myenv):
         # http://lists.llvm.org/pipermail/cfe-dev/2015-November/045852.html
         #
         if env.TargetOSIs('posix') and not env.ToolchainIs('clang') and conf.CheckForFortify():
+            if conf.CheckForFortifyDefined():
+                conf.env.Prepend(
+                    CPPFLAGS='-U_FORTIFY_SOURCE'
```

Comment:
Rather than just having -U_FORTIFY_SOURCE -DFORTIFY_SOURCE, which feels a little strange, would it be better if we just didn't set our own _FORTIFY_SOURCE if the compiler has already done it?

In other words, just write:

```
if !conf.CheckForForifyDefined():
    conf.env.Append(
        CPPDEFINES=[
            ('_FORTIFY_SOURCE', 2)
```